src/SAML2/XML/samlp/StatusDetail.php

```diff
@@ -85,8 +123,10 @@ public function toXML(DOMElement $parent = null): DOMElement
             $parent->appendChild($e);
         }
 
-        if (!empty($this->detail)) {
-            $e->appendChild($e->ownerDocument->importNode($this->detail, true));
+        if (!empty($this->details)) {
```

Comment:
Oh, I can actually comment on this one 😅

Same case as before, the `if` is redundant in this case, as `foreach` tests the condition.